File: backend/src/main/java/com/example/backend/models/DeliveryView.java
package com.example.backend.models;

public class DeliveryView {
    String restaurantName;
    String deliveryPlaced;
    String status;
    String estimatedDelivery;
    String actualDelivery;

    public DeliveryView() {
    }

    public DeliveryView(String restaurantName, String deliveryPlaced, String status, String estimatedDelivery,
            String actualDelivery) {
        this.restaurantName = restaurantName;
        this.deliveryPlaced = deliveryPlaced;
        this.status = status;
        this.estimatedDelivery = estimatedDelivery;
        this.actualDelivery = actualDelivery;
    }

    public String getRestaurantName() {
        return restaurantName;
    }

    public void setRestaurantName(String restaurantName) {
        this.restaurantName = restaurantName;
    }

    public String getDeliveryPlaced() {
        return deliveryPlaced;
    }

    public void setDeliveryPlaced(String deliveryPlaced) {
        this.deliveryPlaced = deliveryPlaced;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getEstimatedDelivery() {
        return estimatedDelivery;
    }

    public void setEstimatedDelivery(String estimatedDelivery) {
        this.estimatedDelivery = estimatedDelivery;
    }

    public String getActualDelivery() {
        return actualDelivery;
    }

    public void setActualDelivery(String actualDelivery) {
        this.actualDelivery = actualDelivery;
    }

}

File: frontend/src/app/models/DeliveryView.ts
// src/app/models/delivery-view.model.ts

export class DeliveryView {
  restaurantName: string;
  deliveryPlaced: string;
  status: string;
  estimatedDelivery: string;
  actualDelivery: string;
  constructor(
    restaurantName: string,
    deliveryPlaced: string,
    status: string,
    estimatedDelivery: string,
    actualDelivery: string
  ) {
    this.restaurantName = restaurantName;
    this.deliveryPlaced = deliveryPlaced;
    this.status = status;
    this.estimatedDelivery = estimatedDelivery;
    this.actualDelivery = actualDelivery;
  }
}
